code/game/objects/items/weapons/melee/misc.dm

```diff
@@ -86,6 +86,18 @@
 	force = 0
 	on = 0
 
+/obj/item/weapon/melee/classic_baton/telescopic/suicide_act(mob/user)
+	var/mob/living/carbon/human/H = user
+	var/obj/item/organ/internal/brain/B = H.getorgan(/obj/item/organ/internal/brain)
+
+	user.visible_message("<span class='suicide'>[user] stuffs the [src] up their nose and presses the 'extend' button! It looks like they're trying to clear their mind.</span>")
+	playsound(loc, 'sound/weapons/batonextend.ogg', 50, 1)
+	sleep(3)

```

Comment:
will remember this next time! good catch. I'm going to just remove those sleeps though, I think